src/scss/components/_badges.scss

```diff
@@ -0,0 +1,42 @@
+/**
+ * Badge Component
+ * =================================
+ * Example – Menu – Restaurant delivery information
+ *
+ * Used for styling small pieces of information.
+ *
+ * Documentation:
+ * https://fozzie.just-eat.com/styleguide/ui-components/custom/badges
+ */
+
+$badge-bgColor    : $grey--offWhite !default;
```

Comment:
Super NIT – this looks slightly off the rest of the indenting below 😞

Reply:
Fixed.